packages/create-plugin/src/types.ts

```diff
@@ -12,6 +12,13 @@ export type CliArgs = {
 
 export type TemplateData = {
   pluginId: string;
+  pluginName: string;
+  pluginDescription: string;
+  hasBackend: boolean;
+  orgName: string;
+  pluginType: PLUGIN_TYPES;
+  hasGithubWorkflows?: boolean;
+  hasGithubLevitateWorkflow?: boolean;
```

Comment:
I think we probably shouldn't have optional properties in the TemplateData type. Maybe I am wrong, but I think looking at it from the template's perspective it's better if the available data properties always have a value - and they are set to a sane default value by `getTemplateData()` in case they are not set by the user prompt.